tools/c7n_azure/c7n_azure/resources/appserviceplan.py

```diff
@@ -77,10 +109,17 @@ class ResizePlan(AzureBaseAction):
     schema = utils.type_schema(
         'resize-plan',
         **{
-            'size': {'type': 'string', 'enum':
+            'size': {'oneOf': [
```

Comment:
We can probably add a method to generate schema for Lookup values.. Smth like

```
'size': lookup_schema({'type': 'string', 'enum': ['F1'...]}),
```

Can be easier to update existing code.